atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -88,24 +93,20 @@ def run(self):
         try:
             self.read_configs()
         except SkipResolveComposesPlugin as abort_exc:
-            # OSBS2 TBD: `override_build_kwarg` is imported from build_orchestrate_build
-            override_build_kwarg(self.workflow, 'yum_repourls', self.repourls, None)
             self.log.info('Aborting plugin execution: %s', abort_exc)
-            return
+            if self.repourls:
+                for arch in self.architectures:
+                    self.yum_repourls[arch].update(self.repourls)
```

Comment:
```suggestion
            for arch in self.architectures:
                self.yum_repourls[arch].update(self.repourls)
```
No need for the conditional, if self.repourls is an empty list it will behave as intended

Reply:
incorporated suggestion